packages/react-devtools-shared/src/devtools/views/DevTools.js

```diff
@@ -127,6 +127,35 @@ export default function DevTools({
     [enabledInspectedElementContextMenu, viewAttributeSourceFunction],
   );
 
+  const devToolsRef = useRef<HTMLElement | null>(null);
+
+  useEffect(() => {
+    const ownerWindow = devToolsRef.current.ownerDocument.defaultView;
```

Comment:
Flow is failing on CI because `devToolsRef.current` could be null. It's looking for a check like:
```js
const devTools = devToolsRef.current;
if (devTools !== null) {
  // ...
}
```